### ISSUE TITLE
yaml file with typo

### ISSUE THREAD

**Issue by MarcelH-Tabeo** [JJBordy/rules#2]
Let's assume, that the YAML file is made by someone from "the business" who accidentally makes a typo, like this: cxustomer.orders[*].price
You wouldn't know, because the program just runs.
I think it should check if the input is something it can work with, or else give useful info that it can't recognise something.

**Comment by JJBordy** [comment]
You refer strictly to user submitting an invalid input path? I think it's a valid point, will not be difficult to implement.

**Comment by MarcelH-Tabeo** [comment]
Yes, a user can make a typo, so if you can detect that, it would be very usable (and user friendly)
